newIDE/app/src/UI/MaterialUISpecificUtil.js

```diff
@@ -49,3 +49,22 @@ export const getMuiCheckboxValue = (element: HTMLElement): boolean => {
   }
   return true;
 };
+
+export const doesPathContainDialog = (
+  path: Array<Element>,
+  options?: { isVisible: true }
```

Comment:
What's the point of passing the options here?
Are we not always checking the dialog is visible?

Reply:
No, the drawer is considered a dialog but it's not always visible. It's used by the in app tutorial orchestrator to know if a tooltip should be hidden or not

Reply:
I meant in the context of this function `doesPathContainDialog`. (you only use it once with isVisible true)

Reply:
I meant to use the same signature as for `isElementADialog` but you're right